src/routes/index.tsx

```diff
@@ -41,6 +42,10 @@ const routes: RouteObject[] = [
         path: 'glad-screen/:confirmationCode',
         element: <AccountCreated />,
       },
+      {
```

Comment:
remove route plz, component will be shown within carousel component

Reply:
- [x] removed routes for Status component